include/spiritless_po/PoParser.h

```diff
@@ -429,8 +429,9 @@ namespace spiritless_po {
                 s += c;
             }
         }
+#ifndef _MSC_VER // warning C4702: unreachable code
         assert(0);
-        return;
+#endif
```

Comment:
The code block, assert() and return, can be removed.
I think it has no worth remaining.

Reply:
Removed completly.